src/todotxt/task.rs

```diff
@@ -22,6 +40,15 @@ pub enum CompletionMode {
     PriorityToTag,
 }
 
+/// How to set completion date on task completion.
+#[derive(PartialEq, Debug, Clone, Copy)]
+pub enum CompletionDateMode {
+    /// Only add completion date if task has create date is specified.
```

Comment:
Nit: I am not sure how to interpret `if task has create date is specified`. To me, it should be either `if task has creation date` or `if task creation date is specified`.